.travis.yml

```diff
@@ -62,5 +62,9 @@ after_success:
     then
       git fetch --all && git config --global user.email "-" && git config --global user.name "-" && ./gradlew run -Dargs="--since d1"
       npm install -g surge
+      npm install -g markbind-cli
+      cd docs
+      markbind build
+      cd ..
```

Comment:
Optional suggestion, not sure but should work on Travis-CI :thinking::

```diff
- cd docs
- markbind build
- cd ..
+ (cd docs && markbind build)
```
This way you don't need to `cd` back as only the subshell's `pwd` is modified.

Reply:
seems to work fine!